skbio/stats/distance/_permdisp.py

```diff
@@ -0,0 +1,269 @@
+        IDs in `distance_matrix` (i.e., the row labels must be distance matrix
+        IDs), but the order of IDs between `distance_matrix` and the
+        ``DataFrame`` need not be the same. All IDs in the distance matrix must
+        be present in the ``DataFrame``. Extra IDs in the ``DataFrame`` are
+        allowed (they are ignored in the calculations).
+    column : str, optional
+        Column name to use as the grouping vector if `grouping` is a
+        ``DataFrame``. Must be provided if `grouping` is a ``DataFrame``.
+        Cannot be provided if `grouping` is 1-D ``array_like``.
+    test : {'centroid', 'median'}
+        determines whether the analysis is done using centroid or spaitial
+        median.
+    permutations : int, optional
+        Number of permutations to use when assessing statistical
+        significance. Must be greater than or equal to zero. If zero,
+        statistical significance calculations will be skipped and the p-value
+        will be ``np.nan``.
+
+    Returns
+    -------
+    pandas.Series
+        Results of the statistical test, including ``test statistic`` and
+        ``p-value``.
+
+    Raises
+    ------
+    TypeError
+        If, when using the spatial median test, the pcoa ordination is not of
+        type np.float32 or np.float64, the spatial median function will fail
+        and the centroid test should be used instead
+    ValueError
+        If the test is not centroid or median. test is set to median by default
+    TypeError
+        If the distance matrix is not an instance of a distance matrix
+
+    See Also
+    --------
+    permanova
+    anosim
+
+    Notes
+    -----
+    See [1]_ for the original method reference, as well as
+    ``vegan::betadisper``, available in R's vegan package [2]_.
+
+    References
+    ----------
+    .. [1] Anderson, Marti J. "Distance-Based Tests for Homogeneity of
+        Multivariate Dispersions." Biometrics 62 (2006):245-253
+
+    .. [2] http://cran.r-project.org/web/packages/vegan/index.html
+
+    Examples
+    --------
+    Load a 6x6 distance matrix and grouping vector denoting 2 groups of
+    objects:
+
+    >>> from skbio import DistanceMatrix
+    >>> dm = DistanceMatrix([[0,    0.5,  0.75, 1, 0.66, 0.33],
+    ...                       [0.5,  0,    0.25, 0.33, 0.77, 0.61],
+    ...                       [0.75, 0.25, 0,    0.1, 0.44, 0.55],
+    ...                       [1,    0.33, 0.1,  0, 0.75, 0.88],
+    ...                       [0.66, 0.77, 0.44, 0.75, 0, 0.77],
+    ...                       [0.33, 0.61, 0.55, 0.88, 0.77, 0]],
+    ...                       ['s1', 's2', 's3', 's4', 's5', 's6'])
+    >>> grouping = ['G1', 'G1', 'G1', 'G2', 'G2', 'G2']
+
+    Run PERMDISP using 99 permutations to caluculate the p-value:
+
+    >>> from skbio.stats.distance import permdisp
+    >>> import numpy as np
+    >>> #make output deterministic; not necessary for normal use
```

Comment:
Please make the following change each time it comes up in this doc string:

*not necessary for normal use* -> *should not be included during normal use*

(I think that statement should be a little stronger, since it really would technically be wrong for the user to run this a bunch of times with the same seed.)